### DISCUSSION TITLE
Filecoin Ecosystem Collective

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#195] [General]
Are you interested in joining and supporting the Filecoin ecosystem? Is your project built on top of the Filecoin ecosystem?  Are you researching how to leverage or improve IPFS/Filecoin? Submit it here if you want it to be featured in the [Filecoin Ecosystem Grid](https://github.com/filecoin-project/community/discussions/194)? Please include:
- Project/Team/Event name
- Logo
- Category: applications, tools, capitals, R&Ds..?
- Short description
- Link to your website and social media if available 

*We will include selected projects and collaborators in the grid in the upcoming iteration. Please only use this discussion for project/team posting.

**Comment by laurarenpanda** [comment]
- Project/Team/Event name: FileDrive
- Logo
![FileDrive Logo 大尺寸](https://user-images.githubusercontent.com/74699939/124038652-10351180-da34-11eb-84f8-9b0dca969913.png)
- Category: applications/ tools
- Short description
FileDrive has been in the Filecoin ecosystem since 2019. We currently concentrate on solutions of distributed storage based on IPFS and Filecoin Network, and technology development to contribute to the Filecoin Community. As for Slingshot Competition, we have been participating from Slingshot Phase 1 last year and stored over 4 PiB useful data, from several public datasets all over the world. Now, we have already developed a data prep tool Go-Graphsplit, and the first FIL+ dashboard filplus.info. 
- Link to your website and social media if available
FIL+ Dashboard: https://filplus.info
Go-Graphsplit: https://github.com/filedrive-team/go-graphsplit
Twitter: https://twitter.com/FileDrive1
WeChat Official Account: [FileDrive](https://mp.weixin.qq.com/mp/profile_ext?action=home&__biz=MzkzMDE4ODg0MQ==&scene=124#wechat_redirect)

**Comment by flyworker** [comment]
-  Project/Team/Event name  FilSwan
-   Logo 
![FILSWAN logo](https://user-images.githubusercontent.com/8363795/124980847-e4e98e00-e002-11eb-8529-e755f952c99b.png)


-   Category: applications, tools, capitals, R&Ds..
  Reputation System, consumer face apps,developer tooling.
-   Short description
  Filswan provides a decentralized edge infrastructure marketplace and software that moves data storage and analysis as close as necessary on Web 3.0.
  The features currently completed and open to the public is deal biding,miner reputation, IPFS caching, S3 compatible storage sharing and backup to Filecoin network, payment gateway.
-   Link to your website and social media if available
https://www.filswan.com
https://twitter.com/filswan1

**Comment by Calvin-Zhang-Ti** [comment]
- Capital Name: Ti Capital

- Logo

![Ti Capital LOGO](https://github.com/Calvin-Zhang-Ti/Ti-Capital/raw/main/Ti%20Capital%20LOGO.png)


- Category:
Capitals, Miner

- Short description:
Ti Capital is a professional blockchain-based venture capital. We invest in open finance, Web3.0, distributed cloud storage and visionary teams with global ambitions.

Ti Capital 's Filecoin Ecosystem Fund will focus on the Filecoin ecosystem. Together with Filecoin, we recognize the immense possibilities and believe that this collaboration will bring more entrepreneurs and developers into the Filecoin Ecosystem.

- Links
https://twitter.com/capital_ti
https://www.linkedin.com/company/ti-capital-digital-asset/

**Reply by duckie-han** [reply]
Hi @Calvin-Zhang-Ti, i will use your logo from your linkedin profile as the one attached - the link is broken.

**Comment by IPFStarry** [comment]
- Project/Team/Event name: IPFStarry

- Logo
![logo2](https://user-images.githubusercontent.com/86825490/124243550-98103e00-db50-11eb-8416-833ab8f57cb1.jpg)

- Category: applications, tools, capitals, R&Ds..
- Short description
IPFStarry——A blockchain integrated technology service provider focusing on the development and application of distributed storage and web3.0 technology

Wuhan IPFStarry Technologies is an international technology company focused on blockchain technology, distributed storage and R&D. The major business of its branch companies are IPFS mining machine R&D and hosting, mining field construction, mining pool development, IPFS investment and incubation, etc. IPFStarry’s products are IPFStarry research institute, Swarm research institute, Starry Pool, Starry Data, etc. The former headquarters was in Singapore. The team is composed of experts in software, operation and maintenance, hardware and chip technology research and development from Amazon Cloud, Google, NVIDIA, Ali, Tencent and other Internet and chip giants.

The IPFStarry mining fields are deployed all over the world, with the development strategy of “multi-party joint construction as the mainstay, self-construction as a supplement," and currently covers China, Hong Kong (region), Taiwan (region), Singapore, the United States, Germany, and Japan , South Korea, Australia and other countries and regions. At present, it has more than 150 IDC data center resources on a global scale, distributed in four major states of Asia, Europe, North America, and Oceania. The platform has sufficient supply of computing power. It has signed cooperation agreements with many top IDC companies. The computing power is supplied by ten global data bases. All the computing resources are real, and users could visit the computing power center in person.

- Link to your website and social media if available
Official website：www.ipfstarry.com
WeChat Official Account: IPFS繁星科技；IPFS繁星研究院

**Comment by ghost** [comment]
Project Name: HashMix

Logo:
![HashMixLogo](https://user-images.githubusercontent.com/86829888/124256265-afa1f380-db5d-11eb-8895-80510b1f03ef.jpeg)

Category: Applications

Description:
HashMix proposes a tokenization scheme suitable for arbitrary hash power, implements a set of universal decentralized cloud hash power trading, lending, and swapping protocols to solve hash power fraud, hash power oversold, over-simplified product design, and lack of liquidity and flexibility in the current market, enables the free redistribution of global hash power.

The first product HashMix will launch is a Filecoin lending platform that miners can borrow FIL by pledging their storage power and FIL holders can earn a considerable interest rate by making a deposit.

Website and social media:
Website: [hashmix.org](https://hashmix.org)
Twitter: [@HashMixOfficial](https://twitter.com/HashMixOfficial)
Telegram: [HashMixChannel](https://t.me/HashMixChannel
)

**Reply by ghost** [reply]
Thanks, excited to find HashMix on the wall! But can we change to a higher resolution logo? 

![hashmix_logo_black_vertical](https://user-images.githubusercontent.com/86829888/126290014-821f2f63-d271-4975-be75-c74f6de97ecd.png)

**Reply by ghost** [reply]
@jennijuju

**Reply by duckie-han** [reply]
We will upgrade the overall reso of the whole grid in the next update,

**Comment by AthenaExplorer** [comment]
> Project/Team/Event name

Athena Explorer

> Logo

![雅典娜英文版logo](https://user-images.githubusercontent.com/85341169/124431372-da30be00-dda2-11eb-8259-13d7fd388330.png)

> Category: applications, tools, capitals, R&Ds..?

applications/tools/R&D

> Short description

An exclusive function based on technical algorithms and driven by data, exists to fill gaps in the industry, and to provide industry participants with in-depth industry analysis, node analysis, output calculation, information in front fields and operator selecting services.

> Link to your website and social media if available

Website: https://www.atpool.com/en-US/
WeChat Official Account: AthenaExplorer
Github: @AthenaExplorer
Slack: @AthenaExplorer
Twitter: @AthenaExplorer

**Comment by MatrixStorage** [comment]
- Project/Team/Event name
Matrix Storage

- Logo
![image](https://user-images.githubusercontent.com/82867405/124299488-0fb18d80-db90-11eb-8cf3-1a71c3fdcbca.png)

- Category: applications, tools, capitals, R&Ds..?
applications, tools, R&D

- Short description
Matrix Storage possesses of easy to use PaaS services, OpenAPI services and storage toolsets, to provide customers with stable and reliable decentralized storage services based on IPFS network.

- Link to your website and social media if available
Website: https://storage.anmaicloud.com/
WeChat Official Account: MatrixStorage
Twitter：MatrixStorage
Github：MatrixStorage
 Slack：矩阵存储MatrixStorage

**Comment by MatrixStorage** [comment]
- Project/Team/Event name
Easy NFT

- Logo
![NFT](https://user-images.githubusercontent.com/82867405/124301058-dda12b00-db91-11eb-9486-fff2a8971d1b.png)

- Category: applications, tools, capitals, R&Ds..?
applications, tools, R&D

- Short description
Web3.0 oriented. With support for connecting to multiple applications and automated task platforms, It enables NFT developers to easily connect to IPFS network.

- Link to your website and social media if available
https://github.com/MatrixStorageTech2021/easynft

**Comment by IPFSUnion** [comment]
- Project/Team/Event name：Xingchi Cloud

- Logo：

![星驰云LOGO定稿6 30-1_画板 1 副本](https://user-images.githubusercontent.com/68763845/124431286-c1280d00-dda2-11eb-828f-9c3c8c9cf2b6.png)

- Category: applications, tools

- Short description：
Xingchi Cloud is committed to becoming the world's leading distributed storage cloud service provider. As the first cloud service platform launched by a Filecoin industry company, Xingchi Cloud will help traditional industries store a large amount of industry real data and effectively promote the application space of the entire distributed storage industry. Xingchi Cloud has strong product research and development capabilities and excellent customer service, and is committed to fully driving the digital future with data technology and empowering all walks of life to enter the data age.

- Link to your website and social media if available
Website: http://www.filcloud.com/

**Comment by liyunzhi-666** [comment]
- Project/Team/Event name:  TJ Wallet 

- Logo:
![image](https://user-images.githubusercontent.com/52949274/124580092-5bab4d80-de82-11eb-962c-325b90ff16e1.png)

- Category: wallets

- Short description: Professional, Safe and Credible Digital Asset Solution.Perfect wallet ecology, offline secure storage and management of your digital assets.Support for 40+ Coins included BTC, ETH, FILECOIN and 2000+ Tokens.Multi-signature Authentication--Import multi-signature addresses, sign and send multi-signature transactions.
DApp Ecosystem--Access ETH, EOS, TRON, and other blockchains with more than 3000 high-quality DApps

- Link to your website and social media if available: 
https://en.tjwallet.net/
https://twitter.com/TJ_Wallet

**Comment by haiyaqingdao** [comment]
- Project/Team/Event name: DappOnline

- Logo:


![image](https://user-images.githubusercontent.com/32318074/124582423-92826300-de84-11eb-94f9-2f5989630219.png)


- Category: applications, tools

- Short description: The world's leading blockchain ecology includes 4000+ blockchain ecological applications and data from dozens of public chains such as eth, EOS, Tron, steem, Iost, ont and so on.DappOnline is the world's leading blockchain application (DApp) store where you can find the latest and greatest DApp information and data every day. We will provide DApp development, DApp news, DApp data, DApp promotion and other services. We hope to promote the ecological development of DApp and let people discover the value brought by blockchain application. Looking forward to your participation, let us witness the arrival of the next generation Internet.

- Link to your website and social media if available: 
  https://dapponline.io/
  https://twitter.com/DappOnline

**Comment by cabinvc** [comment]
Project/Team/Event name： Cabin VC
Logo： 
![5a814d167c82c749f21f76e2841123d](https://user-images.githubusercontent.com/87113631/124874901-dbf4be80-dffa-11eb-99a4-077cab9c85f0.jpg)


Category: capitals, R&Ds，Miner
Short description：We are participating in the wave of digital change of blockchain technology to real society via public welfare,investing and buliding industrial ecology.
We involved in filecoin since 2019, invested in many distributed storage projects. We have also established hardware ecosystem department, focusing on filecoin mining and study on other hardware mining.  

Link to website and social media ：https://www.cabin.vc/
                                                         https://twitter.com/CabinVC

**Comment by EvaYueh** [comment]
Project/Team/Event name: eyeM5 2100CS

Logo: 

![eyeball logo](https://user-images.githubusercontent.com/86940683/124892894-d86a3300-e00c-11eb-9611-639575a7b651.JPG)

Category: Infrastructure

Short description: 
eyeball FinTech Co., Ltd., located in Taiwan where is well-known for its high quality, high density HPC server manufacturing, is specialized in dedicated HPC server design and development by implementing specific algorithm(s) for a specific application scenario. Filecoin mining HPC server is one of our products using the most powerful AMD 7F72 CPU with 64 bit, 48 core, 3.9 GHz specifications, Samsung 1024 G DRAM 3200, Samsung 6.4TB*1 SSD, and 450w*2 TDP to ensure competitive hash rate and high storage capacity. It is proved that eyeM5 2100CS Filecoin HPC server has high yield of FiL by a set of totally 9 machines focusing on worker, acceleration, and storage together with 2 supporting machines for administration and monitoring, not to mention the features in terms of high IDC space utilization, low power consumption, and low latency. Due to outstanding revised version of Filecoin mining software the lucky value is ranged between 400-1000. The quality and delivery on-time rate of Filecoin machine can be guaranteed by a sound supply chain partnership. 

Link to website and social media: 
http://www.eyem5.com/en-index.html

**Comment by jhookersyd** [comment]
Name: Holon Innovations

Category: Ecosystem Capital 

Short description: Holon Innovations is the trusted bridge into the global digitalised economy, we invest in next-generation digital infrastructure that is reshaping the world. We're driving innovation in collaboration with business, technology, academic and social impact partners, consulting government and regulators to provide a “whole of system” approach.

Website:  https://holon.investments/innovations/

Logo:
[Holon Global Innovations Final.pdf](https://github.com/filecoin-project/community/files/6798068/Holon.Global.Innovations.Final.pdf)

**Comment by duckie-han** [comment]
**Version Updated as of 12 July 2021**

![image](https://user-images.githubusercontent.com/84303468/125321559-662c8200-e36f-11eb-9e76-ee1092911fc9.png)

**Reply by liyunzhi-666** [reply]
Why does my product "TJ Wallet" not appear on the latest map? @duckie-han

**Reply by flyworker** [reply]
Why filswan as a project get two dev grants for the most people asked feature S3 compatible to filecoin. Team of market retrieval builder not listed as a ecosystem project?

**Reply by haiyaqingdao** [reply]
I submitted an application (DappOnline), but it is not shown in the picture.

**Comment by AthenaExplorer** [comment]
Dear Sir,
This is Athena Explorer Team.
Could you please enlarge our Logo and place it in a larger space(like after
NBFS)?
Currently, our Logo is put at the bottom edges and doesn't look like the
original image.😂
Here, I send again our logo for you to replace.
Thank you so much.

duckiehan ***@***.***> 于2021年7月13日周二 上午12:16写道：

**Reply by duckie-han** [reply]
Hi AthenaExplorer, I will update this in the next iteration that we send out. Thank you.

**Comment by MatrixStorage** [comment]
您好，矩阵存储、EASYNFT的LOGO在展示上有点小问题，麻烦调整一下
- 矩阵存储：LOGO上下没有显示完全
- EASYNFT：左右没有显示完全
以下为具体图示，烦请处理，谢谢：
![image](https://user-images.githubusercontent.com/82867405/125386050-3496d380-e3ce-11eb-97dd-4780025e0b67.png)
--------
另外⬇️，是重新提供的LOGO图片
<img width="515" alt="MatrixStorage" src="https://user-images.githubusercontent.com/82867405/125386917-ade2f600-e3cf-11eb-9329-38a8cf04ab77.png">
![EASYNFT](https://user-images.githubusercontent.com/82867405/125386922-b0455000-e3cf-11eb-874c-f34a778f0018.png)

**Reply by duckie-han** [reply]
Hi Matrix Storage, I will update this in the next iteration that we send out. Thank you for pointing this out.

**Comment by alexLizhenyu** [comment]
* Project/Team/Event name: StorageLabs
* Logo
![logo](https://user-images.githubusercontent.com/37353847/125389591-0a481480-e3d4-11eb-88ca-8ac109a532db.png)


* Category: applications/ tools
* Short description
At StorageLabs we focus on the development of IPFS ecosystem, our mission is to build a massive data trading platform where users can store, retrieve and edit their data at exabytes scale. We are currently building a mirror library of Docker images with functionalities such as docker image storage, caching, retrieval, and indexing.
* Link to your website and social media if available
FIL+ Dashboard: http://datalabs.storagelabs.io

**Comment by crossfidao** [comment]
Project/Team/Event name :CrossFi

Logo ：![Aaron Swartz](https://www.crossfimain.com/crossfidao_files/logo.png)

Category: applications

Short description
CrossFi is a cross-chain liquidity sharing protocol characterized by multi-chain lending and synthetic assets. It aims to connect and release all isolated liquidity of existing public chains, and is committed to breaking technical barriers of all mainstream public chain assets through leading interoperability technologies. 
 
With CrossFi, users and DeFi applications on different chains will be able to communicate with each other quickly and safely, without having to consider technical details. Everyone hopes to get all independent DeFi liquidity with a simple click, this will be the future of CrossFi
 
CrossFi will be the first to support Filecoin lending. CrossFi is committed to uniting more top miners, formulating Filecoin industry standards, and greatly improving the efficiency of the Filecoin ecosystem's financial services. The goal is to solve the liquidity pressure of 30% of the Filecoin ecosystem.

Link to your website and social media if available
https://www.crossfimain.com
https://twitter.com/globalcrossfi

**Comment by maxvint** [comment]
- Project/Team/Event name: CoinSummer
- Logo
  
![CoinSummer-Logo-Color](https://user-images.githubusercontent.com/1715211/126281561-3230628d-efed-4fcd-85e9-eef04d3f70f9.png)

- Category: Ecosystem Collaborators
- Short description
  CoinSummer team is committed to the major cryptocurrency and blockchain projects, and has developed multiple applications and technical research on Bitcoin/Lightning Network, Ethereum, Filecoin and Polkadot.

- Link to your website and social media if available
  - CoinSummer:  [https://coinsummer.io](https://coinsummer.io)
  - CoinSummer TV: [https://video.coinsummer.io](https://video.coinsummer.io)
  - Filecoin Community China: [https://github.com/filecoin-project/community-china](https://github.com/filecoin-project/community-china)

**Reply by duckie-han** [reply]
Hi @yuwenhui! just a note that coinsummer was already inside the grid under ecosystem collaborators so does not required to be added.

**Comment by ShuGary** [comment]
* Project/Team/Event name: 点存科技（DCTech）
* logo
* 
![资源 5@10x-8](https://user-images.githubusercontent.com/52781726/126284717-b4c06cc0-8fe6-4f8a-aeaf-fae7545f5115.png)


* Category: applications, tools, capitals, R&Ds..
* Short description
Diancun technology is committed to building a world-leading distributed storage technology platform.

Since 2018, Diancun technology has provided enterprises with professional distributed storage and computing services, software development, operation and maintenance hosting and other services to create a distributed storage industry ecosystem from providing computing equipment to providing computing capabilities.

Diancun technology has also successfully used the Filecoin network to help traditional industries store massive amounts of real data. The main service industries include biocomputing, video surveillance, gene sequencing, video live broadcast, etc., effectively promoting the application space of the entire distributed storage industry.

Diancun technology always adheres to the business tenet of "leading technology, first-class products and high-quality service", adheres to the service concept of "customer success > company Success", and is committed to contributing to the sustainable development of ecological prosperity in the field of distributed storage and computing.
* Link to your website and social media if available
   Website: www.diancun.net
   WeChat Official Account:ipfsdiancun（点存科技）
   Twitter:https://twitter.com/DiancunTeach

**Reply by apolloo123** [reply]
DIgiCode Tech业界良心企业

**Comment by EvaYueh** [comment]
The filecoin-project, below, had been uploaded to the community 12 days
ago, but I haven't seen eyeball's logo in the category of the Filecoin
Ecosystem. Please kindly help locate the logo at your earliest convenience.

Best regards,

Eva Yi-Yuan Yueh

Project/Team/Event name: eyeM5 2100CS

Logo:

[image: eyeball logo]
<https://user-images.githubusercontent.com/86940683/124892894-d86a3300-e00c-11eb-9611-639575a7b651.JPG>

Category: Infrastructure

Short description:
eyeball FinTech Co., Ltd., located in Taiwan where is well-known for its
high quality, high density HPC server manufacturing, is specialized in
dedicated HPC server design and development by implementing specific
algorithm(s) for a specific application scenario. Filecoin mining HPC
server is one of our products using the most powerful AMD 7F72 CPU with 64
bit, 48 core, 3.9 GHz specifications, Samsung 1024 G DRAM 3200, Samsung
6.4TB*1 SSD, and 450w*2 TDP to ensure competitive hash rate and high
storage capacity. It is proved that eyeM5 2100CS Filecoin HPC server has
high yield of FiL by a set of totally 9 machines focusing on worker,
acceleration, and storage together with 2 supporting machines for
administration and monitoring, not to mention the features in terms of high
IDC space utilization, low power consumption, and low latency. Due to the
outstanding revised version of Filecoin mining software the lucky value is
between 400-1000. The quality and delivery on-time rate of Filecoin
machines can be guaranteed by a sound supply chain partnership.

Link to website and social media:
http://www.eyem5.com/en-index.html

Jonathan Hooker ***@***.***> 於 2021年7月12日 週一 上午9:34寫道：

**Comment by FayWang54** [comment]
- Project name: FiveToken
- Logo:
![FiveToken](https://user-images.githubusercontent.com/79350091/126472431-87caed3e-0e91-4858-8c14-f64f56b51f86.JPG)

- Category: application & tool
- Short description:
 Bridge Filecoin ecosystem to Web 3 metauniverse and help explore Web 3 DApps with highly-integrated display platform, reliable ID verification and bridge tool across chains.

- Link to your website and social media：
  Website: https://fivetoken.io/ 
  Twitter: https://twitter.com/fivetokenio

**Comment by monna52** [comment]
- Project/Team/Event name: SiliconGo

- Logo
![2  硅格半导体完整logo+R标_透明背景](https://user-images.githubusercontent.com/87857229/126771583-90ec6c8c-418a-4640-80f0-c9e3d4f13f82.png)

- Category: tools

- Short description
SiliconGo provides industry customers with one-stop storage solutions including storage controllers, industrial storage modules, IP and design services, and its products cover consumer, enterprise, industrial, and automotive applications .
With 14 years of technology accumulation and business expansion, SiliconGo has established a complete storage product line of general storage (USB/SD), embedded storage (UFS/eMMC/SPI-NAND) and SSD storage (SATA/PCIe), it’s cumulative shipments exceed one billion sets.
SiliconGo has obtained 203 authorized invention patents, owns the core IP of all storage controllers, and provides integrated storage and computing solutions to meet the challenges of emerging applications such as AIoT and data centers.

- Link to your website and social media if available
Website: www.yeestor.com

**Comment by ohash-ai** [comment]
- Project name : FILWallet

- Logo: 
![Filwallet logo](https://user-images.githubusercontent.com/12134762/126762628-7d618526-7abb-4fd0-901c-cec531b3f0ca.png)

- Category:  
    applications & tools

- Short description
        As a decentralized digital asset wallet which focusing on the Foincoin, FIL WALLET is easy-used and convenient with basic full functions. It has Multi-sig Wallet which is for security, and Dalayed Function which can help your large amount transfer safer. The safest, the most professional wallet for Filecoin players.

- Link to your website and social media if available
     Website: https://filwallet.io/
     Twitter: https://twitter.com/filwalletio

**Comment by duckie-han** [comment]
**Hi all, 
Please see the updated ecosystem grid as of 27 July 2021**. 
![1920x1080 aspect - alt title position (3)](https://user-images.githubusercontent.com/84303468/127116037-27479c0a-ab88-4566-9062-ef4a0450254d.png)

**Reply by ShuGary** [reply]
Why does my product not appear on the latest map;what does eye emoji mean@duckie-han

**Reply by Joss-Hua** [reply]
:rainbow:    It is exciting that there are more and more ecological participants.

:raised_hands:   If you need the Venus logo, you can use this version.

![Venus logo](https://user-images.githubusercontent.com/75723122/142160196-e59cf64a-191e-40de-9500-c4c88101a775.png)

**Comment by Destore2023** [comment]
Project/Team/Event name
ByteBase Privacy Storage Platform

Logo
![白底方logo](https://user-images.githubusercontent.com/46444977/127178520-49b9c62a-03c7-47f6-921f-720e5757b257.png)

Category: applications, tools, capitals, R&Ds..?
applications, tools, R&D

Short description
ByteBase Privacy Storage Platform provides easy-to-use SaaS services and OpenAPI services. We connect both Data Providers and Miners. Data Providers enjoy stable and reliable decentralized storage services based on IPFS networks and Miners enjoy the benefits from data storage and retrieval.

Link to your website and social media if available https://mp.weixin.qq.com/s/zOhEWmHf8YUszPc_t6JsfQ 
Website: www.bytebase.cn
WeChat Official Account: 冰塔数科
Slack：ByteBase Offiicial

**Comment by whyrusleeping** [comment]
Team Name: Applications Research Group

Logo:
<img width="291" alt="image" src="https://user-images.githubusercontent.com/1243164/127450962-ab76aafd-866c-4460-b567-eb4c33d6132a.png">

Category:
Applications, tooling, R&D

Short Description:
The Application Research Group (ARG) is a team that aims to bolsters the mission by focusing on shipping tools that provide daily utility on the Internet.

Website: https://arg.protocol.ai

**Comment by SFIL-Filecoin** [comment]
Project/Team/Event name:SFIL

Logo：
![微信图片_20210730112457](https://user-images.githubusercontent.com/88187805/127634151-5d057c1c-e1ab-41de-94dd-3c9d551496fd.png)

Category: applications/ tools

Category:  Hashrate Token

Short description：
The First Filecoin Standard Full Hashrate Token

The SFIL Foundation (Decentralization Tech Foundation LTD.Singapore ) owns, initiates and sponsors the SFIL ecosystem.

SFILhas created a broader Filecoin hashrate market which enables free and efficient circulation of compliant money worldwide. Every SFIL is pegged to 0.01T Filecoin full hashrate. Once you hold SFIL in the wallet, you can get free FIL each day. Additionally, it is a token that has no liquidation risks and high-value leverage capability in the secondary market.

Link to your website and social media if available：
Contacts
Official Website: www.sfil.io
Twitter: https://twitter.com/sfil_io
Telegram: https://t.me/sfil_cn
ChainNews Column: SFIL
Weibo: https://weibo.com/u/7567519275?is_hot=1
WeChat Official Account: SFIL Foundation
WeChat Community Account:SFIL Standard Full Hashrate Token

**Comment by peter726** [comment]
Project/Team/Event name：DeFIL
Logo 
![image](https://user-images.githubusercontent.com/58384068/127797656-433e1537-04fd-47f4-b0af-a8c120c4d4cc.png)
Category: Blockchain Technology, Decentralized Storage (IPFS), Decentralized Finance, Applications, Capitals, R&Ds

Short description
DeFIL, a Decentralized Finance Platform layered on top of multichain, creates Standard Computing Power Token("FILST") and provides Lending Services for Filecoin Ecosystem.It was launched on Etherum on Nov 20, 2020. DeFIL has built its strong foundation as being a secure and efficient platform through 8 months of operation. 

The amount of FIL deposited on DeFIL had reached 2 million, which worth close to 400 million US dollars.；

DFL, token issued by DeFIL is available on Gate io, ZB, AEX and many other exchanges.；

DeFIL is funded by many famous token funds, including IPFS Ecological Foundation, FBG Capital, Fenbushi Captial, Monolith Capital, ZB Capital, ZBG Capital and more.

The invention of "FILST" has liquidised the traditional computing power of Filecoin mining. It carries the most crucial value of DeFIL 2.0 and it has provided multiple ways for investors to gain profits.
1、Borrowing by Pledging：Pledging FILST allows borrowing relative amoun of FIL or USDT
2、Mining by Staking：Staking FILST obtains mining output of FIL backed by equal computing power (1FILST=1G)
3、Become LP to earn DFL：Become LP by providing pairs either FILST-USDT or FIL-FILST and pledge in Dex to earn DFL rewards.

Link to your website and social media if available
Website: https://defil.org/
Twitter: @DeFIL
Telegram: https://t.me/defilorg

**Comment by peng1428** [comment]
Project/Team/Event name: DMEX

Logo:
![资源 1@4x](https://user-images.githubusercontent.com/36099932/128284834-c5ca8d14-646c-4883-a6bf-6c04c27297a5.png)

Category: Applications

Description:
DMEX is a decentralized mining power financial service platform. DMEX endeavors to lower the entry barrier for the average users, improve the capital utilization rate for the mining machine sellers, and increase the value of idle assets for the miners. DMEX is building a strong community of shared interests, shared risks, and shared mining incomes for mining machine sellers, miners, and users. The platform has a rigorous risk management system fortified with DAO community governance and smart contract. DMEX will invigorate the mining industry with innovative DeFi products such as the mining power marketplace, mining power backed collateral loans, and mining insurance.

Link to your website and social media if available：
Website：https://www.dmex.finance/
Twitter：https://twitter.com/DMEX_finance
Telegram：https://t.me/DMEX_Chat
Medium: https://medium.com/@dmex

**Comment by EvaYueh** [comment]
Hello Jonathan,

I need to clarify the process of adding a logo to the Filecoin Ecosystem
Grid. Referring to my previous email calling your kind attention to my
inquiry. Please advise by return at your earliest convenience.

The filecoin-project, below, had been uploaded to the community almost a
month ago, but I haven't seen eyeball's logo in the category of
Infrastructure of the Filecoin Ecosystem. Please kindly help locate the
logo at your earliest convenience.

Best regards,

Eva Yi-Yuan Yueh

Project/Team/Event name: eyeM5 2100CS

Logo:

[image: eyeball logo]
<https://user-images.githubusercontent.com/86940683/124892894-d86a3300-e00c-11eb-9611-639575a7b651.JPG>

Category: Infrastructure

Short description:
eyeball FinTech Co., Ltd., located in Taiwan, which is well-known for its
high quality, high density HPC server manufacturing, is specialized in
dedicated HPC server design and development by implementing specific
algorithm(s) for a specific application scenario. Filecoin mining HPC
server is one of our products using the most powerful AMD 7F72 CPU with 64
bit, 48 core, 3.9 GHz specifications, Samsung 1024 G DRAM 3200, Samsung
6.4TB*1 SSD, and 450w*2 TDP to ensure competitive hash rate and high
storage capacity. It is proved that eyeM5 2100CS Filecoin HPC server has
high yield of FiL by a set of totally 9 machines focusing on worker,
acceleration, and storage together with 2 supporting machines for
administration and monitoring, not to mention the features in terms of high
IDC space utilization, low power consumption, and low latency. Due to the
outstanding revised version of Filecoin mining software the lucky value is
between 400-1000. The quality and delivery on-time rate of Filecoin
machines can be guaranteed by a sound supply chain partnership.

Link to website and social media:
http://www.eyem5.com/en-index.html

Jonathan Hooker ***@***.***> 於 2021年7月12日 週一 上午9:34寫道：

Jonathan Hooker ***@***.***> 於 2021年7月12日 週一 上午9:34寫道：

**Comment by birdcagelml** [comment]
- Project/Team/Event name:  Fgas

- Logo

![image](https://raw.githubusercontent.com/birdcagelml/images/master/logo-dark.png)


- Category: 
applications, tools

- Short description
Fgas provides reliable real-time mining costs analysis, gas fees monitoring, mining account reconciliation and input-output calculator  based on the Filecoin Economy and technical algorithms.


- Link to your website and social media if available
Website: https://fgas.io/

**Comment by longfeiWan9** [comment]
@jennijuju The pic of [Matrix](https://matrix.org/) under the messaging category is not the right one, this is the correct one;

![matrix](https://user-images.githubusercontent.com/34025001/128476657-5562d6b1-e7c6-4147-969f-0ef66e5806cc.png)

**Reply by PuskitOfficial** [reply]
@jennijuju Hi, this is Pluskit. Below is our information.

Project/Team/Event name：Pluskit

![缂栫粍](https://user-images.githubusercontent.com/93764859/141721344-99a83a4a-e42c-4e47-b945-668344ada79f.png)

Category：tools

Short description：Fil+ ecosystem all-rounded data tool

Link to your website and social media if available：
https://pluskit.io/
https://twitter.com/pluskitofficial

**Comment by seong-k** [comment]
>  Project/Team/Event name : 
- TVCC NEWS

>  Logo : 

![ez5hnk5coa](https://user-images.githubusercontent.com/87347053/144184618-3744811c-7f7e-4bb3-9cef-b59f030f1b03.png)

> Category(applications, tools, capitals, R&Ds...?) : 
- Media Platform

>  Short description : 

TVCC, the leader in sharing news and information on cryptocurrency and blockchain located in South Korea, is a media company that strives to deliver accurate information. TVCC creates the latest news specifically about the cryptocurrency and blockchain industry.  Also, we are developing media content to introduce recent trends and issues to audiences about IPFS and Filecoin. Moreover, we are saving the content that TVCC has created in Filecoin. Furthermore, all processes listed so far will be created into video content for introduction to our subscribers.

> Link to your website and social media if available :

-  https://tvcc.kr

-  https://www.youtube.com/c/TVCC-Broadcas

**Comment by sxxfuture-official** [comment]
**Data Cold Storage**
- Project/Team/Event name : SXX Future Data - Data Cold Storage
- Logo : 
![image](https://user-images.githubusercontent.com/113963951/201296445-d897222b-56b3-48c3-83e9-cc58b763d974.png)

 
- Category: applications, tools
- Short description：Data Cold Storage is a data disaster recovery and cold storage platform developed based on Web3/IPFS, serving scenarios such as government, institutions and large enterprises. Through Data Cold Storage, users can upload data with long-term preservation needs to Data Cold Storage IPFS, which becomes highly secure, low-cost and encryptable uplinked data by sealing the storage to Filecoin for cold storage. These deposited data can be seen through the official IPFS browser and can also be queried at Filecoin for data information.
- Link to your website and social media if available :
1. https://twitter.com/sxxfuture
2. https://github.com/sxxfuture-official
3. https://www.sxxfuture.com/

**Web3 Cloud Disk**
- Project/Team/Event name : SXX Future Data - Web3 Cloud Disk
- Logo : 
![image](https://user-images.githubusercontent.com/113963951/201296508-6cdd080f-67ed-49fb-bf60-f250a10b93be.png)

 
- Category: applications, tools
- Short description：A decentralized file storage application for the Web3 era, based on blockchain technology and IPFS (InterPlanetary File System) decentralized distributed storage technology, it can provide massive storage space for users who require data security and decentralization, achieve data trust, multi-node backup, decentralized storage, fast retrieval and other functions It is suitable for data backup, data distribution, collaborative sharing, NFT storage, etc. in the Web3 era.
- Link to your website and social media if available :
1. https://twitter.com/sxxfuture
2. https://github.com/sxxfuture-official
3. https://www.sxxfuture.com/

**Comment by Irmagogo** [comment]
Hi there, @jennijuju 
Is this discussion still active?
